app/forms/support_interface/conditions_form.rb

```diff
@@ -0,0 +1,85 @@
+module SupportInterface
+  class ConditionsForm
+    class OfferConditionField
+      include ActiveModel::Model
+
+      attr_accessor :id, :text
+
+      validates :text, length: { maximum: MakeAnOffer::MAX_CONDITION_LENGTH, too_long: ->(c, _) { "Condition #{c.id + 1} must be %{count} characters or fewer" } }
+    end
+
+    include ActiveModel::Model
+
+    attr_accessor :application_choice, :standard_conditions, :further_conditions, :audit_comment
+
+    MIN_FURTHER_CONDITIONS = 4
```

Comment:
With the new changes in manage, there is no minimum on the number of conditions set.